src/core/control/LatexController.cpp

```diff
@@ -115,16 +117,16 @@ void LatexController::findSelectedTexElement() {
         // this will get the position of the Latex properly
         EditSelection* theSelection = control->getWindow()->getXournal()->getSelection();
         xoj::util::Rectangle<double> rect = theSelection->getSnappedBounds();
-        this->posx = rect.x;
-        this->posy = rect.y;
+        this->posx = static_cast<int>(rect.x);
+        this->posy = static_cast<int>(rect.y);
```

Comment:
All those static_casts feel off to me. Bluntly casting should never happen: we probably want to std::round, std::floor or std::ceil those values before.
In this case, I don't know which one it should be. This needs understanding the class a bit better.

Reply:
I this case the posx, posy, imgWidth, imgHeight should be double. They are used to in method call on TexImage there expect double.

Reply:
By changing the attributes to double the static_cast is mostly avoided.